app/models/course/discussion/post.rb

```diff
@@ -1,5 +1,11 @@
 class Course::Discussion::Post < ActiveRecord::Base
   acts_as_forest order: :created_at
 
+  after_initialize :set_topic, if: :new_record?
```

Comment:
@allenwq do we set defaults after initialize or before validation or what...? We changed the default a few weeks ago right?